rust/Makefile.toml

```
@@ -0,0 +1,79 @@
+[tasks.format-rust]
+args = ["fmt"]
+command = "cargo"
+description = "Format rust sources files."
+install_crate = { rustup_component_name = "rustfmt" }
+
+[tasks.lint-rust-format]
+args = ["fmt", "--all", "--", "--check"]
+command = "cargo"
+description = "Check format of sources files."
+install_crate = { rustup_component_name = "rustfmt" }
+
+[tasks.lint-rust]
+args = [
+  "clippy",
+  "--workspace",
+  "--locked",
+  "--all-targets",
+  "--all-features",
+  "--",
+  "-D",
+  "clippy::all",
+  "-D",
+  "warnings",
+]
+command = "cargo"
+description = "Check lint of all sources files."
+install_crate = { rustup_component_name = "clippy" }
+
+[tasks.lint-toml]
+args = ["lint"]
+command = "taplo"
+description = "Check lint of all toml files."
+install_crate = { crate_name = "taplo-cli", binary = "taplo", test_arg = "--help" }
+
+[tasks.format-toml]
+args = ["fmt"]
+command = "taplo"
+description = "Format toml file"
+install_crate = { crate_name = "taplo-cli", binary = "taplo", test_arg = "--help" }
+
+[tasks.lint]
+dependencies = ["lint-rust-format", "lint-rust", "lint-toml"]
+
+[tasks.format]
+dependencies = ["format-rust", "format-toml"]
+
+[tasks.clean]
+args = ["clean"]
+command = "cargo"
+
+[tasks.build]
+args = ["run"]
+command = "cargo"
```

I suggest you to let this task as the [template-rust](https://github.com/okp4/template-rust/blob/main/Makefile.toml#L52) do for the build task and create a new task to run the proto generation. 
```suggestion
[tasks.build]
args = ["build", "--release", "--locked"]
command = "cargo"

[tasks.generate-proto]
args = ["run"]
command = "cargo"
```